integration_tests/e2e/refer.cy.ts

```diff
@@ -342,4 +342,81 @@ context('Refer', () => {
     const completePage = Page.verifyOnPage(CompletePage)
     completePage.shouldContainPanel('Referral complete')
   })
+
+  describe('Submitting a referral', () => {
+    const course = courseFactory.build()
+    const courseOffering = courseOfferingFactory.build()
+
+    const prisoner = prisonerFactory.build({
+      firstName: 'Del',
+      lastName: 'Hatton',
+    })
+    const person = personFactory.build({
+      currentPrison: prisoner.prisonName,
+      name: 'Del Hatton',
+      prisonNumber: prisoner.prisonerNumber,
+    })
+
+    const prison = prisonFactory.build({ prisonId: courseOffering.organisationId })
+    const organisation = OrganisationUtils.organisationFromPrison('an-ID', prison)
+
+    const referral = referralFactory.build({
+      offeringId: courseOffering.id,
+      prisonNumber: person.prisonNumber,
+      status: 'referral_submitted',
+    })
+
+    beforeEach(() => {
+      cy.signIn()
+
+      cy.task('stubCourseByOffering', { course, courseOfferingId: courseOffering.id })
+      cy.task('stubCourseOffering', { courseId: course.id, courseOffering })
+      cy.task('stubPrison', prison)
+      cy.task('stubPrisoner', prisoner)
+      cy.task('stubReferral', referral)
+      cy.task('stubUpdateReferralStatus', referral.id)
+
+      const path = referPaths.checkAnswers({ referralId: referral.id })
+      cy.visit(path)
+    })
+
+    it('Shows an error when the user tries to submit a referral without confirming the details', () => {
```

Comment:
We only have one other example of this (find: 'Viewing a single offering'), but I think we have preference for no initial capital when `it` is nested in `describe`